lib/train/extras/os_common.rb

```diff
@@ -96,6 +98,8 @@ def detect_family
       # TODO: extend base implementation for detecting the family type
       # to Windows and others
       case uname_s
+      when /unrecognized command verb/
```

Comment:
problem is that if we omit this part, it matches 

 when /./
        @platform[:family] = 'unix'